### PR TITLE
feat(web): display candidate and assessment name dynamically on the Interview Recording Page

### LINKED ISSUE #1012: fix(web): assessment name/candidate name is all hardcoded

### PR DESCRIPTION
**Pull request by Thomas-Boi**
1. Candidate and Assessment name are loaded from the interview data
  a. Sensible defaults are provided while waiting:
    - Default candidate name is `Welcome!`. Loaded name is `Welcome, Thomas!`. Exclamation mark can be removed if wanted.
    - Default assessment name is `Assessment`. Loaded name is `My Assessment Name`.
2. Commented out the timer code.

Fixes #1012

### CODE REVIEW

File: web/src/template/InterviewRecordingPage/InterviewRecordingPage.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ interface InterviewRecordingPageTemplateProps {
   progressbarCount: number;
   totalQuizCount: number;
   quiz: Quiz;
+  interview: Interview | undefined;
```
Comment (kokiebisu) [inline web/src/template/InterviewRecordingPage/InterviewRecordingPage.tsx:27]: Would
```
interview?: Interview 
```
look cleaner?

File: web/src/screens/InterviewRecordingPage.tsx [hunk 2]
```diff
@@ -58,13 +59,14 @@ const InterviewRecordingPage = () => {
       );
       if (response.data.status === 'success') {
         // caseSensitiveAxiosInstance doesn't work
-        const interview = response.data.payload.interview;
-        interview.quizzes.map((quiz: any) => {
+        const theInterview = response.data.payload.interview;
```
Comment (kokiebisu) [inline web/src/screens/InterviewRecordingPage.tsx:62]: Would interviewPayload be more descriptive?

**Review by kokiebisu — APPROVED** [review]
Feel free to update based on my comments if you feel they're better

File: web/src/template/InterviewRecordingPage/InterviewRecordingPage.tsx (hunk 1 above)
Comment (Thomas-Boi) [reply web/src/template/InterviewRecordingPage/InterviewRecordingPage.tsx:27]: Agreed, I forgot using undefined counts as it being optional as well 👍

File: web/src/screens/InterviewRecordingPage.tsx (hunk 2 above)
Comment (Thomas-Boi) [reply web/src/screens/InterviewRecordingPage.tsx:62]: That's a better name for sure 👍

**Pull request merged**